website/source/api/health.html.markdown

```diff
@@ -61,7 +61,8 @@ $ curl \
     "Notes": "",
     "Output": "",
     "ServiceID": "",
-    "ServiceName": ""
+    "ServiceName": "",
+    "ServiceTags": null
```

Comment:
At which level do you want to provide empty arrays instead of `nil`? memdb, catalog response, agent response ?